mitmproxy/tcp.py

```diff
@@ -9,16 +9,19 @@
 class TCPMessage(serializable.Serializable):
 
     def __init__(self, from_client, content, timestamp=None):
-        self.content = content
         self.from_client = from_client
+        self.content = content
         self.timestamp = timestamp or time.time()
 
     @classmethod
     def from_state(cls, state):
-        return cls(*state)
```

Comment:
All other objects return a dict - it seemed inconsistent to return a tuple here.